src/components/ui/Input.jsx

```diff
@@ -23,7 +23,7 @@ const Input = (props) => {
         <>
             {label && (
                 <label
-                    className='font-normal  mt-1 whitespace-nowrap flex:me-10 flex:self-center'
+                    className='  mt-1 whitespace-wrap flex:me-10 text-light-gray/80 font-light flex:self-center'
                     htmlFor={rest.id || rest.name}
                 >
```

Comment:
(rest) prop is optional and it is not known what its value will be.
Maybe this prop is not passed, so we will have this kind of error
TypeError: Cannot read properties of undefined (reading 'name')
TypeError: Cannot read properties of undefined (reading 'id')

Therefore, it cannot be used as a value in the htmlFor
Not to mention that one of the rules of accessibility is that it should be the same value as the id of the input
So you better make me like this 
```suggestion
                    htmlFor={id}
                >
```